src/providers/ArticleProvider.jsx

```diff
@@ -1,21 +1,21 @@
-import { useEffect, useState } from "react";
+import { useEffect } from "react";
 import Articles from "../pages/Articles";
 import { getArticles } from "../utils/api";
-import axios from 'axios';
+import TopicList from "../components/TopicList";
 
-function ArticleProvider() {
-const [articles, setArticles] = useState([]);
-useEffect(() => {
-    getArticles()
-    .then(({articles}) => {
-        setArticles(articles)
-    })
-}, [])
+function ArticleProvider({ articles, setArticles }) {
```

Comment:
I think calling this a Provider is misleading as that's a specific thing in React. I would maybe call it ArticlesPage or similar?